mep/books/tests/test_books_signals.py

```diff
@@ -0,0 +1,121 @@
+from unittest.mock import patch
+
+from parasolr.django.indexing import ModelIndexable
+import pytest
+
+from mep.books.models import Creator, CreatorType, Work, WorkSignalHandlers
+from mep.people.models import Person
+
+
+@pytest.mark.django_db
+@patch.object(ModelIndexable, 'index_items')
+def test_creatortype_save(mock_indexitems):
+    new_type = CreatorType(name='Typesetter')
+    # unsaved - ignored
+    WorkSignalHandlers.creatortype_save(CreatorType, new_type)
+    mock_indexitems.assert_not_called()
+
+    author_type = CreatorType.objects.get(name='Author')
+    # unsaved - ignore
+    # saved but no associated books
```

Comment:
seems like only one of these comments can be true?

Reply:
looks like the unsaved ignore is extra